template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+# Good Docs Project - Template contributing guide
+
+Welcome to the Good Docs Project!
+If you’re reading this guide, that probably means you’d like to get involved and start contributing to the project.
+This document should hopefully help you become a full templateer (which is how we refer to the members of our community).
+
+One of the core missions of the Good Docs Project is to provide high-quality documentation templates to open source software projects and beyond. However, we also engage in many other similar initiatives around docs advocacy, docs education, and docs tooling.
+We value all contributions to the Good Docs Project initiatives, including templates and our other initiatives. The Join the community section of this guide explains how to get involved in those other related initiatives and provides links for more information.
+
+## Table of contents
+
+- [Before you start](#before-you-start)
+  - [Template roles and resources](#template-roles-and-resources)
+  - [Required deliverables](#required-deliverables)
+- [Overview of the template writing phases](#overview-of-the-template-writing-phases)
+- [Join the community](#join-the-community)
+- [Adopt a template](#adopt-a-template)
+  - [Guidelines for choosing a template](#guidelines-for-choosing-a-template)
+- [Research and draft the template](#research-and-draft-the-template)
+  - [Recommended research strategies](#recommended-research-strategies)
+- [Workshop the draft of the template](#workshop-the-draft-of-the-template)
+  - [Accepting feedback from others](#accepting-feedback-from-others)
+- [Submit a pull request](#submit-a-pull-request)
+- [Improve the template with user feedback](#improve-the-template-with-user-feedback)
+
+## Before you start
+
+Before starting, ensure you are familiar with The Good Docs Project’s goals, key concepts, and supported personas. For more information, see [About the Good Docs Project](https://thegooddocsproject.dev/about/).

```

Comment:
This link does not appear to have any information about personas. If we are arguing that someone needs to be familiar with our personas before reading this, we should link directly to info about the personas. I'm not convinced, however, that knowing about our personas is a requirement for reading through the contributing process. That's probably something that can be introduced to new members in the writing working groups. Or we can link to it as part of the contributing guide.

Reply:
I don't think we need to say anything about personas at this point. I'm not missing any context by not knowing about personas.

Reply:
Fair enough. I'll remove "personas."

Reply:
I've adopted this suggestion and I'm resolving this comment.